src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -823,6 +835,33 @@ static uct_md_ops_t md_ops = {
     .detect_memory_type = uct_cuda_copy_md_detect_memory_type
 };
 
+static ucs_status_t uct_cuda_copy_md_check_is_ctx_set_flags_supported(void)
+{
+    static ucs_status_t status = UCS_ERR_LAST;
```

Comment:
Do you think that it would be simpler to return 0/1 since the function is called only during md open?
Then it will return 0 when compiling with CUDA_VERSION < 12000.